recipes/pantools/build.sh

```diff
@@ -2,11 +2,12 @@
 
 set -eu -o pipefail
 
-PACKAGE_HOME=$PREFIX/share/$PKG_NAME
-mkdir -p $PREFIX/bin
-mkdir -p $PACKAGE_HOME
-
-cp -R dist/* $PACKAGE_HOME/
-cp $RECIPE_DIR/pantools.py $PACKAGE_HOME/pantools.py
-chmod +x $PACKAGE_HOME/pantools.py
-ln -s $PACKAGE_HOME/pantools.py $PREFIX/bin/pantools
+PACKAGE_HOME="${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}"
+mkdir -p "${PREFIX}/bin"
+mkdir -p "${PACKAGE_HOME}"
+
+cp -R dist/* "${PACKAGE_HOME}/"
+cp "${RECIPE_DIR}/pantools.py" "${PACKAGE_HOME}/pantools.py"
+
+ln -s "${PACKAGE_HOME}/pantools.py" "${PREFIX}/bin/pantools"
```

Comment:
For some reason the final package _does not_ create a symlink to the `${PREFIX}/bin/`, but a hardlink. This causes the wrapper script to being unable to find `pantools.jar`, resulting in build failure.